docs/application/dotnet/guides/connectivity/webview.md

```diff
@@ -0,0 +1,165 @@
+# WebView
+
+The WebView features include accessing Web pages and Web content in your application.
+
+[Tizen.WebView](https://samsung.github.io/TizenFX/latest/api/Tizen.WebView.html) namespace covers various features for Web browsing, such as loading and displaying Web pages and navigating through the browsing history.
+
+Use the Web features to create a simple Web browser:
+
+1. [Initialize and finalize WebView](#initialize)
+
+2. [Create a WebView object](#create).
+
+3. [Load Web page](#load).
+
+4. [Handle loading events](#event).
+
+5. [Set Cookie options](#cookie).
+
+## Prerequisites
+
+To enable your application to use WebView functionality:
+
+1. To use `Tizen.WebView` namespace, create `ElmSharp-Beta` project using [Project Widzard](https://developer.tizen.org/development/visual-studio-tools-tizen/tools/project-wizard) of Visual Studio.
+
+2. To use class of `Tizen.WebView` namespace, the application has to request permission by adding the following privileges to the `tizen-manifest.xml` file:
```

Comment:
2. To use the class of the Tizen.WebView` namespace, you must request for permission by adding the following privileges to the `tizen-manifest.xml` file, as shown in the following code snippet:

Reply:
Done.